tests/users/createCart.test.ts

```diff
@@ -350,21 +349,19 @@ describe('POST /users/current/carts', () => {
 
     const coach = users.find((findUser) => findUser.type === UserType.coach);
     const attendant = users.find((findUser) => findUser.type === UserType.attendant);
-    const spectator = users.find((findUser) => findUser.type === UserType.spectator);
 
     expect(body.url).to.startWith(env.etupay.url);
 
-    // player place + player reduced price + coach place + attendant place + spectator place + 4 * ethernet-7
-    expect(body.price).to.be.equal(2500 + 2000 + 1500 + 1500 + 1200 + 4 * 1000);
+    // player place + player reduced price + coach place + attendant place + 4 * ethernet-7
+    expect(body.price).to.be.equal(2000 + 1500 + 1200 + 1200 + 4 * 1000);
```

Comment:
T'as rebase (ou équivalent) sur la dev où on a mis à jour les prix 😉 
```suggestion
    expect(body.price).to.be.equal(2500 + 2000 + 1500 + 1500 + 4 * 1000);
```